caps/repo.go

```diff
@@ -81,6 +81,14 @@ func (r *Repository) HasType(t *Type) bool {
 	return false
 }
 
+// HasType checks if the repository contains the given type.
```

Comment:
Noted, I'll fix this in the next round. I have a few more Foo() == foo() + locking methods in the pipe